app/controllers/grantees_controller.rb

```diff
@@ -1,8 +1,10 @@
+require "api_delegator"
+
 class GranteesController < ApplicationController
   # GET /grantees/1 or /grantees/1.json
   def show
     @grantee = Grantee.new(
-      hses_grantee: Api::FakeData::Grantee.new(id: params[:id]).data
+      hses_grantee: ApiDelegator.use("hses", "grantee", {id: params[:id]}).request.data
```

Comment:
[suggestion] feels a little weird for the controller to know about the api delegator (and `hses`). could this be in the model so the controller just has to pass the `id` param as usual?